packages/headless/src/api/knowledge/stream-answer-api.ts

```diff
@@ -169,53 +187,60 @@ export const answerApi = answerSlice.injectEndpoints({
           organizationId,
           environment
         );
-        const insightGenerateEndpoint = `${platformEndpoint}/rest/organizations/${organizationId}/insight/v1/configs/${insightConfiguration?.insightId}/answer/${generatedAnswer.answerConfigurationId}/generate`;
-        const generateEndpoint = `${platformEndpoint}/rest/organizations/${organizationId}/answer/v1/configs/${generatedAnswer.answerConfigurationId}/generate`;
-        await fetchEventSource(
-          insightConfiguration ? insightGenerateEndpoint : generateEndpoint,
-          {
-            method: 'POST',
-            body: JSON.stringify(args),
-            headers: {
-              Authorization: `Bearer ${accessToken}`,
-              Accept: 'application/json',
-              'Content-Type': 'application/json',
-              'Accept-Encoding': '*',
-            },
-            fetch,
-            onopen: async (res) => {
-              const answerId = res.headers.get('x-answer-id');
-              if (answerId) {
-                updateCachedData((draft) => {
-                  draft.answerId = answerId;
-                });
-              }
-            },
-            onmessage: (event) => {
-              updateCachedData((draft) => {
-                updateCacheWithEvent(event, draft, dispatch);
-              });
-            },
-            onerror: (error) => {
-              throw error;
-            },
-            onclose: () => {
+        const answerEndpoint = insightConfiguration
+          ? buildAnswerEndpoint(
+              platformEndpoint,
+              organizationId,
+              generatedAnswer.answerConfigurationId!,
+              insightConfiguration.insightId
+            )
+          : buildAnswerEndpoint(
+              platformEndpoint,
+              organizationId,
+              generatedAnswer.answerConfigurationId!
+            );
+
+        await fetchEventSource(answerEndpoint, {
+          method: 'POST',
```

Comment:
Was this you or is it prettier formatting it like that??

```suggestion
        await fetchEventSource(
          answerEndpoint, 
          {
            method: 'POST',
            ...
```